core/cap-0038.md

```diff
@@ -968,6 +978,11 @@ x = ceil[Xy / (Y - y) / (1 - F)] .
 In this proposal, `F = 0.003` which corresponds to 0.3% (this is encoded by
 `LIQUIDITY_POOL_FEE_V18`).
 
+#### RevokeSponsorshipOp
+This proposal adds a new result code `REVOKE_SPONSORSHIP_MALFORMED` for
+`RevokeSponsorshipOp`. This result code will be returned on validation if
+`RevokeSponsorshipOp.ledgerKey().type() == ASSET_TYPE_POOL_SHARE`.
```

Comment:
Think this needs to say `.type() = TRUSTLINE && RevokeSponsorshipOp.ledgerKey().trustLine().asset.type() == ASSET_TYPE_POOL_SHARE`.

Reply:
I don't think that's right. The pool share trustline can be sponsored. This should actually say `RevokeSponsorshipOp.ledgerKey().type() == LIQUIDITY_POOL` though.

Reply:
Yes, that's the right change.